src/main/java/org/hibernate/infra/bot/EditPullRequestBodyAddIssueLinks.java

```diff
@@ -0,0 +1,155 @@
+package org.hibernate.infra.bot;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Set;
+
+import org.hibernate.infra.bot.config.DeploymentConfig;
+import org.hibernate.infra.bot.config.RepositoryConfig;
+import org.hibernate.infra.bot.util.CommitMessages;
+
+import org.jboss.logging.Logger;
+
+import io.quarkiverse.githubapp.ConfigFile;
+import io.quarkiverse.githubapp.event.CheckRun;
+import io.quarkiverse.githubapp.event.CheckSuite;
+import io.quarkiverse.githubapp.event.PullRequest;
+import jakarta.inject.Inject;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHIssueState;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHPullRequestCommitDetail;
+import org.kohsuke.github.GHRepository;
+
+/**
+ * @author Marco Belladelli
+ */
+public class EditPullRequestBodyAddIssueLinks {
+	private static final Logger LOG = Logger.getLogger( EditPullRequestBodyAddIssueLinks.class );
+
+	private static final String START_MARKER = "<!-- Hibernate GitHub Bot issue links start -->";
+
+	private static final String END_MARKER = "<!-- Hibernate GitHub Bot issue links end -->";
+
+	private static final String LINK_TEMPLATE = "https://hibernate.atlassian.net/browse/%s";
+
+	@Inject
+	DeploymentConfig deploymentConfig;
+
+	void pullRequestChanged(
+			@PullRequest.Opened @PullRequest.Reopened @PullRequest.Edited @PullRequest.Synchronize
```

Comment:
```suggestion
			@PullRequest.Opened @PullRequest.Reopened @PullRequest.Edited
```

Reply:
Isn't this needed? AFAIU the synchronize event is triggered when a user updates the tracking branch of a PR, so if for example he added a new commit with a new issue key we should add that to the description too.

Reply:
Ah right, sorry, I forgot you were extracting info from commits. My bad.